lib/aptible/cli/helpers/tunnel.rb

```diff
@@ -8,6 +9,20 @@ class Tunnel
         def initialize(env, ssh_cmd)
           @env = env
           @ssh_cmd = ssh_cmd
+
+          # The :new_pgroup key specifies the CREATE_NEW_PROCESS_GROUP flag for
+          # CreateProcessW() in the Windows API. This is a Windows only option.
+          # true means the new process is the root process of the new process
+          # group.
+          # This flag is necessary for Process.kill(:SIGINT, pid) on the
+          # subprocess.
+          if RUBY_PLATFORM =~ /mswin|mingw/i
+            @stop_signal = :SIGINT
+            @windows_opts = { new_pgroup: true }
+          else
+            @stop_signal = :SIGHUP
+            @windows_opts = {}
+          end
         end
```

Comment:
These are all constants as far as the program is concerned, I don't think they should be defined in the `#initialize` method since they'll be strictly identical for all instances.

I can see two options here:

- Either define them as constants behind a conditional (I'd rename `windows_opts` to e.g. `spawn_opts` though, since they're only Windows options if we're on Windows..!)
- Or create two `Tunnel` subclasses — one for Windows and one for POSIXish.

The latter *might* be a little overkill here considering there are actually few differences, but that's up to you to decide! 😄